spec/requests/employee_invitation_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'EmployeeInvitationController', type: :request do
+  let(:user) { create(:user, :admin) }
+  let(:not_listed_user) { create(:user) }
+  let(:company) { create(:company) }
+  let(:headers) { { Authorization: sign_in(user) } }
+
+  describe 'POST /companies/:company_id/employee_invitations' do
+    context 'when user is no registered on Comarev' do
+      subject(:invitation_controller) do
+        post company_employee_invitation_path(company), params: { email: 'test_email@test.com' },
+          headers: headers
+      end
+
+      it 'Returns status ok' do
```

Comment:
it doesn't need to be capitalized
```suggestion
      it 'returns status ok' do
```